<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSRepoPathWizardPage</class>
 <widget class="QWizardPage" name="MSRepoPathWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item row="0" column="2">
      <widget class="QLineEdit" name="repoURLEdit">
       <property name="inputMethodHints">
        <set>Qt::ImhUrlCharactersOnly</set>
       </property>
       <property name="placeholderText">
        <string>Repository URL (i.e. https://github.com/my_org/my_repo.git)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="pathEdit">
       <property name="inputMethodHints">
        <set>Qt::ImhUrlCharactersOnly</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Click button to select or create empty folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Repository URL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Clone path</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="selectFolder">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme=":icons/folder.svg"/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="progressEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
